TACC/Core-CMS-Custom
# chore: clean up templates for TACC/Core-CMS#710

## Overview

Clean up templates with a https://github.com/TACC/Core-CMS that supports template overwrite instead of inheritance.

## Related

- requires: https://github.com/TACC/Core-CMS/pull/710
- similar to https://github.com/TACC/Core-CMS-Resources/pull/189

## Changes

- ~~load v3.12 beta~~
	<sup>~~cuz [TACC/Core-CMS#710](https://github.com/TACC/Core-CMS/pull/710) is off of v3.12.0-beta.3~~</sup>
    (merges of `main` make this moot)
- <del>remove favicon load from `assets_custom`</del>
	<sup><del>cuz it is now loaded in `assets_core`</del></sup>
  <ins>update `assets_custom` to match latest Core-CMS code</ins>
	<sup><ins>and clarify "TODO" task</ins></sup>
- remove `demdata_cms/…/templates/nav_search.raw.html`
	<sup>cuz it is never loaded anyway (`INCLUDES_SEARCH_BAR = False`)</sup>

## Testing

> [!IMPORTANT]
> **Skipped**, because changeset has become negligible.

1. Deploy [all sites](https://github.com/TACC/Core-CMS-Custom/tree/main).
2. Verify favicon matches production.
3. Verify page titles match production.

## UI

N/A
